docs/processes/release/tasks/publish/index.md

```diff
@@ -0,0 +1,512 @@
+
+---
+
+## Upgrade LocaleExplorer and other demos/samples
+
+... to the ICU project site.
+
+Build the icuapps module following the README's. Update code and/or docs as
+needed. "Reference" platforms for icuapps are: RedHat Linux and win32. On Linux,
+icuapps is built against the "make install "'ed ICU. So, run ICU4C's configure
+with --prefix=/some/where pointing to where ICU4C should be installed, and also
+follow icuapps's README.
+
+Install the new locale explorer and other demos/samples onto the public demo
+hosting site.
+
+---
+
+## ICU Collation Demo
+
+Update the ICU collation demo's `index.html` with the new ICU version’s
+available collators.
+
+1.  Do a clean build (configure, make clean, make install, make check).
+    1.  Otherwise, the data build may not pick up a new locale into the
+        coll/res_index.txt file.
+2.  Run [icu-demos >
+    webdemo/collation/build.sh](https://github.com/unicode-org/icu-demos/blob/master/webdemo/collation/build.sh)
+    (after modifying it for your system).
+3.  Copy-paste the output `available-collators.txt` into `index.html`.
+    1.  Or, easier: Use a GUI difftool (e.g., meld) to compare the two and move
+        the changes into index.html.
+    2.  `meld webdemo/collation/index.html ../available-collators.txt`
+4.  See for example the changes for
+    [ICU-11355](https://unicode-org.atlassian.net/browse/ICU-11355)
+
+For details see the comments at the start of the build.sh file.
+
+---
+
+## Repository Branch and Tags
+
+⚠ Careful! The following examples contain specific version, revision and ticket
+numbers. Adjust them for the current release! Easiest: Paste into an editor,
+fix, then paste into the terminal.
+
+#### Creating Maintenance Branch.
+
+Sanity check: Update to the latest repository revision. (Master branch if you
+copy from master, maintenance branch if you copy from there.)
+
+```none
+git checkout master 
+git pull upstream master 
+git log -n1 
+commit bcd0... (HEAD -> master, upstream/master, ...)
+```
+
+Ensure that your local branch is in sync with the upstream branch. Make sure you
+are checking the upstream remote, and not your fork!
+
+Build & test ICU4C & ICU4J on your machine.
+
+Create the maintenance branch from the current known good master ref.
+
+```none
+git checkout -b maint/maint-63 
+git push -u upstream maint/maint-63
+```
+
+#### Tagging
+
+Use the GitHub GUI to create both the "release" and the "tag" at the same time:
+
+<https://github.com/unicode-org/icu/releases/new>
+
+Fill in the tag name, such as "release-63-rc" or "release-63-1", and make the
+target the "maint/maint-xx" branch (such as maint/maint-63).
+
+Set the title to "ICU 63 RC" or "ICU 63.1". Fill in the description using the
+text from the announcement email. (You can also look at previous releases and
+possibly re-use some of the generic text, such as links to the API docs, etc.)
+
+Your screen should look like this:
+
+![image](maint-63-rc-screenshot.png){width="320" height="171"}
```

Comment:
The width and height show up as literal text on the rendered page. Incompatible syntax?

Reply:
Done. (Deleted.)